packages/server/server/api/destinations/destination.controller.ts

```diff
@@ -59,6 +61,26 @@ class DestinationController implements Controller {
             })
             .catch((err) => next(err));
     }
+
+    private updateDestinationUserRating(req: Request, res: Response, next: NextFunction) {
+        DestinationService.getDestinationById(req.params.id)
+            .then((dest: Destination) => {
+                const newRatingArray = dest.userRatings.filter(
```

Comment:
Any reason why we're doing filter +  concat instead of simple map? Is it because you want the updated review to be at the end of list?

Reply:
I think I wanted at the end of the list just to make it easier for me to see the update. I will change it to map

Reply:
Oh we can't because it might be a new rating.